packages/hathor-rpc-handler/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export default () => {
```

Comment:
question: Should we have the Hathor header here?

Reply:
Done! Thanks